### DISCUSSION TITLE
Mod Name sorting ability - Also changes load order. What?

### DISCUSSION THREAD

**Discussion by Benadictus** [bcssov/IronyModManager#489] [Ideas]
On the collection mod panel, clicking (even accidentally) on the Mod Name header changes the load order of the mods in that collection - without any warning.  
It is often the case (as with me) that the existing order must be as it is for the game to load and new games or saved games be loaded after Stellaris itself.
As such - Alpha sorting of the names is FINE as long as actually chaning the order requires Something to the effect of "Make this my load order".   Alpha sorting of the "mod titles" should, I believe, retain the "Order" numbers they had before sorting.

**Comment by bcssov** [comment]
In Windows Explorer, when you click on the filename or datetime header, it does not give you any warning either. If you accidentally hit the header, you undo the changes in Irony (ctrl+z to undo).

As for the rest, that's why you have multiple collections. Collection for game A, collection for game B, and so on. Is there a point in there that I am missing?

**Reply by Benadictus** [reply]
- well you are, of course, correct as to Windows Explorer - however changing the "view" of a list of files does **not** affect the files content (equivalent to load order) or the directory in which they reside.
- I did not know about <Ctrl>-Z , thank you for that.  More than one level of undo?
- Multiple collections reference is good planning, except I wasn't planning to accidentally click the [Mod Name] title-bar ;)

**Comment by bcssov** [comment]
1. Any software with grid headers behaves in similar way I have no plans to have it ask for permission.
2. Yes, as one would expect from undo\redo

**Comment by alexmannen** [comment]
bumping with my own question is it possible to use AI llm to tell it to read error logs and check the mods to guess the optimal load order.
im way to dumb to sort my 100 mod list myself

**Reply by bcssov** [reply]
TL;DR: No.
LLMs can’t determine mod load order - they don’t understand the actual in-game logic, dependencies, or file-level overrides. That kind of analysis requires full parsing of the game’s scripts and assets, not just text pattern matching.
